Makefile

```diff
@@ -82,7 +82,7 @@ gen-api: ## Generate OpenAPI client into internal/generated/api
 .PHONT: build-docs
 build-docs:
 	go get github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs
-	go run github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs generate --examples-dir ./examples
+	CAPELLA_OPENAPI_SPEC_PATH="$(shell pwd)/openapi.generated.yaml" go run github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs generate --examples-dir ./examples
```

Comment:
will this same command be run when binary is pushed to registry ?

worth trying to do a test release